tests/Tests/500__jupyterhub/test-minimal-image.robot

```diff
@@ -1,76 +1,88 @@
 *** Settings ***
-Force Tags       Smoke  Sanity
-Resource         ../../Resources/ODS.robot
-Resource         ../../Resources/Common.robot
-Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
-Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
-Library          DebugLibrary
-Library          JupyterLibrary
-Suite Setup      Begin Web Test
-Suite Teardown   End Web Test
-
-*** Variables ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             DebugLibrary
+Library             JupyterLibrary
+
+Suite Setup         Begin Web Test
+Suite Teardown      End Web Test
+
+Force Tags          Smoke    Sanity
+
 
 *** Test Cases ***
 Open RHODS Dashboard
-  Wait for RHODS Dashboard to Load
+    Wait for RHODS Dashboard to Load
 
 Can Launch Jupyterhub
-  ${version-check} =  Is RHODS Version Greater Or Equal Than  1.4.0
-  IF  ${version-check}==True
-    Launch JupyterHub From RHODS Dashboard Link
-  ELSE
-    Launch JupyterHub From RHODS Dashboard Dropdown
-  END
+    ${version-check} =    Is RHODS Version Greater Or Equal Than    1.4.0
+    IF    ${version-check}==True
+        Launch JupyterHub From RHODS Dashboard Link
+    ELSE
+        Launch JupyterHub From RHODS Dashboard Dropdown
+    END
 
 Can Login to Jupyterhub
-  Login To Jupyterhub  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
-  ${authorization_required} =  Is Service Account Authorization Required
-  Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
-  Wait Until Page Contains Element  xpath://span[@id='jupyterhub-logo']
+    Login To Jupyterhub    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    ${authorization_required} =    Is Service Account Authorization Required
+    Run Keyword If    ${authorization_required}    Authorize jupyterhub service account
+    Wait Until Page Contains Element    xpath://span[@id='jupyterhub-logo']
 
 Can Spawn Notebook
-  [Tags]  ODS-901  ODS-903
-  Fix Spawner Status
-  Spawn Notebook With Arguments  image=s2i-minimal-notebook
+    [Tags]    ODS-901    ODS-903
+    Fix Spawner Status
+    Spawn Notebook With Arguments    image=s2i-minimal-notebook
+
+Verify jupyterlab server pods are spawned in a custom namespace
+    [Documentation]    Verifies that jupyterlab server pods are spawned in a custom namespace (rhods-notebooks)
+    [Tags]    Sanity    
```

Comment:
Do we need sanity. I see force tag has been applied for both smoke and sanity. @lugi0

Reply:
This test suite always had both the Smoke and Sanity tags. We can probably remove Sanity from here since it's already applied by `Force Tags` but I think we can keep both.

Reply:
yes..That was my suggestion

Reply:
@pranavsuryaw can you remove the `Sanity` tag from here?

Reply:
yes, sure